pyanaconda/dbus/structure.py

```diff
@@ -207,17 +207,23 @@ def generate_fields(cls):
     return fields
 
 
-def generate_string_from_data(obj, skip=None):
+def generate_string_from_data(obj, skip=None, add=None):
     """Generate a string representation of a data object.
 
     Set the argument 'skip' to skip attributes with sensitive data.
 
+    Set the argument 'add' to add other values to the string
+    representation. The attributes in the string representation
```

Comment:
I suggest just "sorted alphabetically" as this is just a list of strings which don't have a name.

Reply:
Fixed.